mmcv/runner/runner.py

```diff
@@ -31,6 +31,8 @@ class Runner(object):
         log_level (int): Logging level.
         logger (:obj:`logging.Logger`): Custom logger. If `None`, use the
             default logger.
+        meta (dict): A dict records some import information such as environment
```

Comment:
The type annotation is not correct. It should be `(dict | None)` or `(dict, optional)`